spec/jobs/regular/group_smtp_email_spec.rb

```diff
@@ -85,6 +85,7 @@
 
   it "creates an IncomingEmail record with the correct details to avoid double processing IMAP" do
     subject.execute(args)
+    expect(ActionMailer::Base.deliveries.count).to eq(1)
```

Comment:
The assertion here is pretty generic since any email being sent in the process will result in this test passing. Is there a simple assertion that we test the email we expect is sent? Maybe a test for the subject or template?

Reply:
Yes I can easily add this